dashboard/app/views/home/index.html.haml

```diff
@@ -3,11 +3,21 @@
 = render partial: 'devise/confirmations/alert'
 
 #landingpage
-  .lines_of_code_header
-    %a.lines_of_code_link{href: CDO.code_org_url('/loc')}
-      - loc = fetch_metrics['lines_of_code']
-      - loc = number_with_delimiter(loc, :delimiter => ',')
-      = t('home.lines_of_code', lines_count: loc)
+  .welcome.row
+    .welcomeleft.span2
+      = image_tag('codeorg-studio-logo-white.png', style: "height:120px")
+    .welcomeright.span8
+      .lines_of_code_header
+        %a.lines_of_code_link{href: CDO.code_org_url('/loc')}
```

Comment:
Hash attribute should start with one space after the opening brace<br>Hash attribute should end with one space before the closing brace